ipv8/loader.py

```diff
@@ -118,6 +119,15 @@ class A(CommunityLauncher):
         class A(CommunityLauncher):
            ...
 
+        def MyCommunityClass():
+            from my_module.some_submodule import MyCommunityClass
+            return MyCommunityClass
+           ...
+
+        @overlay(MyCommunityClass)
+        class A(CommunityLauncher):
+            pass
```

Comment:
This new example redefines `MyCommunityClass` and uses `...` differently. This issue is repeated at line 173. Please use something close (or equal) to:

```suggestion
        def my_community_class():
            from my_module.some_submodule import MyCommunityClass
            return MyCommunityClass

        @overlay(my_community_class)
        class A(CommunityLauncher):
            ...
```

Reply:
👌